index.js

```diff
@@ -0,0 +1,16 @@
+let Merger = require('./lib/merger');
+
+let merge = (file) => {
```

Comment:
This function can be simplified:

```suggestion
const merge = async (file) => {
  const merger = new Merger({ delimeter: '\n\n' });
  return merger.processFile(file, true);
}
```